JSTests/test262/expectations.yaml

```diff
@@ -1353,12 +1377,210 @@ test/built-ins/RegExp/prototype/Symbol.search/u-lastindex-advance.js:
+test/built-ins/Temporal/Duration/compare/precision-exact-mathematical-values-1.js:
```

Comment:
We'll need to sort through these, though I can do it in a follow-up if you prefer.